driver/cmd/ibm-spectrum-scale-csi/main.go

```diff
@@ -68,9 +68,9 @@ func main() {
 	level := getLogEnv()
 	logValue := getLogLevel(level)
 	value := getVerboseLevel(level)
-	flag.Set("logtostderr", "false")
-	flag.Set("stderrthreshold", logValue)
-	flag.Set("v", value)
+	err := flag.Set("logtostderr", "false")
+	err = flag.Set("stderrthreshold", logValue)
+	err = flag.Set("v", value)
```

Comment:
Please handle errors separately, though there won't be any error in most of the cases, in case there is an error on line 71 or 72, it will be lost

Reply:
Do we need such elaborate error handling in main method for flag declaration.  We could check for all flag errors in one condition.

Reply:
Yes, I believe so, ideally we should have the errors handled separately, otherwise it is as good as handling error only for last flag.Set as err gets overwritten

Reply:
Now I got the errors in different variables and handled error in singe condition.